core/Archive.php

```diff
@@ -234,6 +234,15 @@ public static function factory(Segment $segment, array $periods, array $idSites,
             $isMultipleDate);
     }
 
+    // TODO: check period has today, not equals
```

Comment:
@diosmosis is this a todo that still needs work? Looking at code it works fine as we only look at today anyway.

Reply:
It's not needed, I must have been thinking of non-day periods, but for some reason we don't skip those archives... oh that's because before if we skipped day, the logic to launch higher periods wouldn't be executed. This will be a problem now correct? If so it's needed.

Reply:
Before we were still launching higher periods that include today @diosmosis

Reply:
Ok, then not needed